Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM gcr.io/google_appengine/base
 # at https://hub.docker.com/r/google/cloud-sdk/~/dockerfile/
 
 ENV \
-	CACHE_BUST=1 \
+	CACHE_BUST=2 \
```

Comment:
Yeah, it's just an (ugly) workaround, because we're not using a version number anywhere in that Dockerfile right now :(

I, at some point, would like to combine this repo with the dockercfg generator and will then take a look at extracting a common base image similar to how we handle the AWS images right now.